SimonOkello/Questioner-V1
# [Chore #setup] Setup directory structure

**What does this PR do?**
Sets up directory structure for Questioner API

**Description of Task to be completed?**
Have the following achieved:
The directory structure should allow for scaling of the API

**How should this be manually tested?**
Clone this repo:
`https://github.com/SimonOkello/Questioner.git`
install `tress` package:
`sudo apt-get install tree`
`cd Questioner-V1`
Check the directory structure using tree command
type `tree`

**Screenshot**

![screenshot from 2019-01-11 21-31-42](https://user-images.githubusercontent.com/28682748/51052644-4f631400-15e8-11e9-8f1f-f6edebdae241.png)
